core/include/webview/detail/backends/win32_edge.hh

```diff
@@ -859,6 +866,9 @@ private:
 
   // Blocks while depleting the run loop of events.
   void deplete_run_loop_event_queue() {
+    // prevent premature run loop triggering of the default window size
+    set_is_size_set(true);
```

Comment:
I'm reading this as:

* The size has been set.
* Loop
* If window isn't visible already then the size has *not* been set anymore.

The flag is also set in `set_size()`, and it looks like the flag could be reset back to `false` unconditionally.

I understand the flag is used to prevent `set_size()` from being called during the temporary event loop but I wonder if some renaming would be doable to make this easier to read.

Reply:
See latest PR for improved semantics and additional backstop.

The scenario is that we are depleting the event queue outside of the main event loop, so we don't want to set a default size **yet** (but the lamda has **already** been dispatched in the class constructor). We thus need to deal with this "future". 
1) `default_size_backstop(true)` flags `m_is_size_set` to true so that the dispatched lamda will execute, but do nothing.
2) Loop. The lamda is removed from the event queue.
3) We evaluate:
[if the user has called `set_size` (`!m_is_window_shown`)]
and 
[if this isn't a destructor call (`!is_destructor`)] 
and then conditionally:
  - reset the backstop and 
  - put the `dispatch_size_default()` lamda back on the event queue.

Reply:
We _could_ ignore this like we do on MacOS (ie. `dispatch_size_default` is called **after** `window_init` in the class constructor), except that Windows also depletes the queue on `bind`, and I feel that this is semantically the correct block to apply the brakes.

I am open to an alternative implementation in `add_user_script_impl`, but that is already a rather dense method to consume.

Reply:
Flip flop, eat my hat...

I have implemented this in `add_user_script_impl` only and removed attendant complexities.
This should reduce unneeded execution of the `dispatch_size_default` inner lamda.

As an aside, in `add_user_script_imp`:
```
while (!done) {
  deplete_run_loop_event_queue();
}
```
is a loop within a loop, which was likely a cause of reciprocal execution of the size lamda (and should now no longer happen).

Reply:
See https://github.com/webview/webview/pull/1249/files#r2007965027